Monika After Story/game/pong.rpy

```diff
@@ -430,28 +604,498 @@ init -1 python in mas_pong:
 
 # dialogue shown right when monika wins
 label mas_pong_dlg_winner:
-    m 1hua "I win~!"
+    # adapts the dialog, depending on the difficulty, game length (bounces), games losses, wins and dialog response.
+    # the order of the dialog is crucial, the first matching condition is chosen, other possible dialog is cancelled in the process.
+    # todo: adapt dialog to affection
+    # todo: add randomized dialog
+        
+        
+    #Player lets Monika win after being asked to go easy on her without hitting the ball
+    if monika_asks_to_go_easy and ball_paddle_bounces == 1:
+        m 2lssdrb "Ahaha.."
+        m 1hsb "This isn't exactly what I had in mind."
+        m 4esb "However, I do appreciate the gesture~"
+        $ monika_asks_to_go_easy = False
+        
+        
+    #Player lets Monika win after being asked to go easy on her without hitting the ball too much
+    if monika_asks_to_go_easy and ball_paddle_bounces <= 9:
```

Comment:
Maybe it's just me, but I think it would be easier to read and go over the code if this was an `elif`.

Reply:
that was a bug.